app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<!--[if ( !IE ) | ( gte IE 9 ) ]><!-->
+<%# render 'dough/shared/social_sharing_icon_sprite' %>
```

Comment:
This is a bit peculiar, but I can see we do this in Frontend as well. Do you know what happens in IE9 without the conditional comments?